CAVEATS.md

```diff
@@ -0,0 +1,232 @@
+<!--
+    Copyright 2014, General Dynamics C4 Systems
+
+    SPDX-License-Identifier: GPL-2.0-only
+-->
+
+# Known caveats in the seL4 API and implementation
+
+## Implementation Correctness
+
+The following seL4 architectures have platforms with a C-level functional
+correctness proof. Proof support for further platforms within these
+architectures is on the roadmap and expected in 2025.
+
+- AArch32: Armv7-a with and without hypervisor extensions, no SMMU, with
+  fast path
+  - Platforms (non-hyp): `sabre` (no FPU), `imx8mm-evk` (with FPU)
+  - Platforms (hyp, no FPU): `tk1`, `exynos5`
+- AArch64: Armv8-a with hypervisor extensions only, no SMMU, with fast path
+  - Platforms: `tx2`
```

Comment:
Too early to say something about the PROVERS platform here?

Reply:
Yes, it's not merged yet, and won't be in time for the release. But we can probably add it next week..